src/pages/jaewonhwang/Login/Login.js

```diff
@@ -0,0 +1,83 @@
+import { Component } from "react"; // named export vs. default export
+import "./Login.js";
+import "./Login.scss";
+import { Link, withRouter } from "react-router-dom";
+
+class LoginJW extends Component {
+  constructor() {
+    super();
+    this.state = {
+      //key : value
+      inputId: "",
+      inputPw: "",
+      disabled: true,
+    };
+  }
+
+  goToMain = () => {
+    this.props.history.push("/Mainjw");
+  };
+
+  handleInput = (e) => {
+    // console.log(event);
+    // console.log(event.target);
+    // console.log(e.target.name);
+    // console.log(e.target.value);
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  render() {
+    // console.log(this.state.inputId);
+    const { inputId } = this.state;
+    const { inputPw } = this.state;
+    return (
+      <div className="loginJW">
+        <div className="main_box">
+          <h3 className="main_text">Westagram</h3>
+          <div className="login_id">
+            <input
+              type="text"
+              placeholder="전화번호, 사용자 이름 또는 이메일"
+              name="inputId"
+              value={inputId}
+              //value={this.state.inputId}
+              onChange={this.handleInput}
+            />
+          </div>
+          <div className="login_pw">
+            <input
+              type="password"
+              placeholder="비밀번호"
+              name="inputPw"
+              value={inputPw}
+              //value={this.state.inputPw}
+              onChange={this.handleInput}
+            />
+          </div>
+          <button
+            className="loginBtn"
+            // className={
+            //   this.state.inputId.includes("@") && this.state.inputPw.length >= 5
+            //     ? "btnActive"
+            //     : "btnDisabled"
+            // }
+            disabled={
+              inputId.includes("@") && inputPw.length >= 5 ? false : true
```

Comment:
```suggestion
              inputId.includes("@") && !inputPw.length >= 5
```
뒷 부분에 삼항연산자 안쓰고 이런식으로 처리할 수 있습니다.

Reply:
Boolean 데이터 타입을 잘 활용합시다!! :)